src/VisualStudio/Core/Def/Shared/VisualStudioImageIdService.cs

```diff
@@ -65,6 +66,8 @@ private ImageId GetImageId(ImmutableArray<string> tags)
                 return GetCompositedImageId(
                     CreateLayer(Glyph.Reference.GetImageMoniker(), virtualXOffset: 1, virtualYOffset: 2),
                     CreateLayer(KnownMonikers.PendingAddNode, virtualWidth: 7, virtualXOffset: -1, virtualYOffset: -2));
+            case Glyph.Copilot:
```

Comment:
Instead of this, the `SparkleNoColor` imageId should likely be added here - https://github.com/dotnet/roslyn/blob/main/src/LanguageServer/Protocol/Extensions/Extensions.KnownImageIds.cs#L15

Then https://github.com/dotnet/roslyn/blob/main/src/LanguageServer/Protocol/Extensions/Extensions.cs#L354 can be updated to map the glyph to the right imageid, and shouldn't need any changes to this file

Reply:
Thanks - just updated